templates/shared_services/airlock_notifier/porter.yaml

```diff
@@ -71,7 +71,7 @@ install:
         tre_url: ${ bundle.parameters.tre_url }
         smtp_server_address: ${ bundle.parameters.smtp_server_address }
         smtp_username: ${ bundle.parameters.smtp_username }
-        smtp_password: ${ bundle.parameters.smtp_password }
+        smtpPassword: ${ bundle.parameters.smtpPassword }
```

Comment:
I'm not sure which option is best...
but you can keep the terraform variable name consistent with the rest of them.

Reply:
done